### ISSUE TITLE
RuntimeError in predictor.explain - dogs and cats demo

### ISSUE THREAD

**Issue by abedkhooli** [amaiya/ktrain#81]
In the dogs and cats demo, everything works well except the predictor.explain which returns a runtime error. Here's some details:
```
predictor.predict_filename('/content/dc/valid/cats/cat.11724.jpg') => ['dogs']
predictor.explain('/content/dc/valid/cats/cat.11724.jpg')
---------------------------------------------------------------------------

RuntimeError                              Traceback (most recent call last)

<ipython-input-45-94cd2561b0f0> in <module>()
----> 1 predictor.explain('/content/dc/valid/cats/cat.11724.jpg')

9 frames

/usr/local/lib/python3.6/dist-packages/tensorflow_core/python/ops/gradients_util.py in _GradientsHelper(ys, xs, grad_ys, name, colocate_gradients_with_ops, gate_gradients, aggregation_method, stop_gradients, unconnected_gradients, src_graph)
    489   """Implementation of gradients()."""
    490   if context.executing_eagerly():
--> 491     raise RuntimeError("tf.gradients is not supported when eager execution "
    492                        "is enabled. Use tf.GradientTape instead.")
    493   if src_graph is None:

RuntimeError: tf.gradients is not supported when eager execution is enabled. Use tf.GradientTape instead.
```
using Keras version: 2.2.4-tf 
ktran: '0.10.1'
eli5: '0.10.1'
stellargraph: '0.8.2'

**Comment by amaiya** [comment]
Thanks for reporting issue.  A workaround for right now is to disable V2 behavior by adding this to top of your notebook or script prior to importing ktrain:
```python
import os
os.environ["DISABLE_V2_BEHAVIOR"]="1" 
```

Doing the above will allow you to use `predictor.explain` on images.

**Comment by abedkhooli** [comment]
Workaround fixes the issue. Some images have invisible explanation, though.  Not sure if color issue or no focus in the frame. 
![image](https://user-images.githubusercontent.com/11407254/76249794-bb70f480-624c-11ea-8aee-2fd44c04b928.png)

**Comment by amaiya** [comment]
Thanks.  That's an issue best asked on the `eli5` GitHub page, as `predictor.explain` is only a thin wrapper to that library, which just simply invokes this: `eli5.show_prediction(model, x)`.

For now, I have added a check in `ImagePredictor.explain` to ensure `DISABLE_V2_BEHAVIOR=True`.

**Comment by abedkhooli** [comment]
https://github.com/TeamHG-Memex/eli5/issues/368

**Comment by amaiya** [comment]
Looks like the issue was resolved in the linked thread - thanks.
